### PR TITLE
1.21.3 changelog

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/telegraf/v1.21/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -10,6 +10,51 @@ menu:
     weight: 10
     parent: About the project
 ---
+## v1.21.3 [2022-1-31]
```
Comment (sjwang90) [inline content/telegraf/v1.21/about_the_project/release-notes-changelog.md:13]: Can we set the date to [2022-1-27]? I wanna keep it aligned with what's in the GitHub Changelog

File: data/products.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ telegraf:
   versions: [v1.9, v1.10, v1.11, v1.12, v1.13, v1.14, v1.15, v1.16, v1.17, v1.18, v1.19, v1.20, v1.21]
   latest: v1.21
   latest_patches:
-    "1.21": 2
+    "1.21": 3S
```
Comment (sjwang90) [inline data/products.yml:38]: I don't know what this is but should it be a `3`?

File: content/telegraf/v1.21/about_the_project/release-notes-changelog.md [hunk 3]
```diff
@@ -10,6 +10,51 @@ menu:
...
+- Update `antchfx/jsonquery` module from 1.1.4 to 1.1.5.
+- Update `prometheus/procfs` module from 0.6.0 to 0.7.3.
+- Update `aws/aws-sdk-go-v2/service/cloudwatchlogs` module from 1.5.2 to 1.12.0.
+- Update `kardianos/service` module from 1.0.0 to 1.2.1.
+- Update `couchbase/go-couchbase` module from 0.1.0 to 0.1.1.
+- Update `pion/dtls/v2` module from 2.0.9 to 2.0.13.
+- Update `containerd/containerd` module to 1.5.9.
+
+### Input plugin updates
+- Execd (`execd`): Resolve a text format parsing error for Prometheus parsing.
```
Comment (pierwill) [inline content/telegraf/v1.21/about_the_project/release-notes-changelog.md:37]: Maybe: 
```suggestion
- Execd (`execd`): Resolve a Promethues text format parsing error.
```

**Review by pierwill — APPROVED** [review]
👍 👍

**Pull request merged**